lean/deps/galois_stdlib/src/galois/init/int.lean

```diff
@@ -0,0 +1,16 @@
+import galois.init.nat
```

Comment:
I think rather than update the Lean 3 files to be Lean 4, we should copy the files to the Lean 4 directory so that we still can run Lean 3 examples while we are in transition.